components/EntireScheme.svelte

```diff
@@ -72,13 +73,25 @@
     // TODO Should we prompt before deleting the current scheme?
     reader.onload = (e) => {
       try {
-        gjScheme.set(JSON.parse(e.target.result));
+        gjScheme.set(backfill(JSON.parse(e.target.result)));
       } catch (err) {
         window.alert(`Couldn't load scheme from a file: ${err}`);
       }
     };
     reader.readAsText(e.target.files[0]);
   }
+
+  function backfill(json) {
```

Comment:
If/when this becomes more complicated, we can split out in some way. Robin mentioned this was also an annoying difference for analysis. Before we bulk analyze schemes, we should just run the same backfill process on all the inputs.